src/modules/storage/TransactionsStorage.ts

```diff
@@ -0,0 +1,162 @@
+        this.db.run(sql, (err: any) =>
+        {
+            if (callback != undefined)
+                callback(err);
+        });
+    }
+
+    /**
+     * Put a transaction to database
+     * @param data a transaction data
+     * @param callback If provided, this function will be called when
+     * the database was finished successfully or when an error occurred.
+     * The first argument is an error object.
+     */
+    public putTransaction (block_height: any, tx: any, tx_index: number, tx_hash: string, callback?: any)
+    {
+        if (
+                (block_height == undefined) ||
+                (tx == undefined) ||
+                (tx_index == undefined) ||
+                (tx_hash == undefined)
+        ) {
+            if (callback != undefined)
+                callback("Parameter validation failed.");
+            return;
+        }
+
+        let sql =
+        `INSERT OR REPLACE INTO transactions
+            (block_height, tx_index, tx_hash, type, inputs_count, outputs_count)
+        VALUES
+            (?, ?, ?, ?, ?, ?)`;
+        this.db.run(sql,
+            [
+                block_height,
+                tx_index,
+                tx_hash,
+                tx.type,
+                tx.inputs.length,
+                tx.outputs.length
+            ], (err: any) =>
+        {
+            if (callback != undefined)
+                callback(err);
+        });
+    }
+
+    /**
+     * Puts all transactions
+     * @param block: block Json data
+     * @param callback If provided, this function will be called when
+     * the database was finished successfully or when an error occurred.
+     * The first argument is an error object.
+     */
+    public putTransactions (block: any, callback?: any)
+    {
+        let idx = 0;
+        let doPut = () =>
+        {
+            if (idx >= block.txs.length)
+            {
+                if (callback != undefined)
+                    callback(null);
+                return;
+            }
+
+            this.putTransaction(block.header.height.value, block.txs[idx], idx, block.merkle_tree[idx], (err: any) =>
+                {
+                    if (!err)
+                    {
+                        idx++;
+                        doPut();
+                    }
+                    else
+                    {
+                        if (callback != undefined)
+                            callback(err);
+                        else
+                            return;
+                    }
+                }
+            );
+        }
+        doPut();
+    }
+
+    /**
+     * Gets a transaction data
+     * @param height the height of the block to get
+     * @param callback If provided, this function will be called when
+     * the database was finished successfully or when an error occurred.
+     * The first argument is an error object.
+     * The second argument is result set.
+     */
+    public getTransactions (height: any, callback?: any)
```

Comment:
Please stop using `any` for the callback and give it a proper type.

Reply:
I changed